README.md

```diff
@@ -109,6 +83,30 @@ c.Invoke(func(map map[string]int, slice []int, array [2]string) {
 })
 ```
 
+### Provide a Constructor
```

Comment:
No prob. I just want the short list to match the order of the longer explanations. I'll adjust the short list instead.